### DISCUSSION TITLE
Survey for understanding communication preferences among MDAnalysis users and developers

### DISCUSSION THREAD

**Discussion by adetutudeborah** [MDAnalysis/mdanalysis#4494] [Polls]
Hello everyone. My name is Adetutu Deborah Oluwasanmi, and I am excited to be a part of this incredible community. Whether you're a seasoned member or a newcomer, your presence adds value to the MDAnalysis community.

We're running a quick survey, initiated by Outreachy applicants, to understand your communication preferences in the MDAnalysis community. Your insights will guide us in enhancing how we connect, share info, and collaborate.

Survey Link: https://forms.gle/NX6tuKKtYP7C3hR97

Kindly take a minute to complete the survey. Thanks.

 Note that this initiative is independent and not officially sanctioned by the MDAnalysis leadership team. Your survey responses will be treated confidentially and used solely for the purpose of this initiative.

**Comment by jennaswa** [comment]
I would like to clarify that this survey is being put out as a self-organized initiative from [Outreachy](https://www.outreachy.org/) applicants. This survey has not been reviewed or approved by the [MDAnalysis project leadership team](https://www.mdanalysis.org/about/#governance). It is therefore not guaranteed that these data will be used to develop an MDAnalysis communication strategy in an official capacity. As such, we are going to close this discussion for the time being.

Jenna Swarthout Goddard (@jennaswa) - MDAnalysis Project, Community and Outreach Manager (on behalf of the MDAnalysis project leadership team)
